# COVID-19: Origin and Pathogenesis

## Abstract
The __COVID-19__ _(stand for __“coronavirus disease”__ and 19 is the year this pandemic started,2019)_ pandemic is among one of the deadliest infectious diseases in recent history`(Morens et al. 2020)`. What makes it different from other viruses  was the specific mechanism for its emergence in humans remains unknown. A virus similar to the SARS COV-2(the pathogen causing the virus) has been discovered in bat of the genus Rhinolophus and in the Sunda pangolin(_Mantis javanica_)`(Desmarest 1822)`. It suggests that animals have played a key role in spreading this outbreak. The first case was noticed in the wet market of Wuhan,China in November 2019.  It is found that viruses can either be directly transmitted from bats(Rhinolophus) or by oral consumption or by the moisture created by the hound of Bat which is likely possible. But since it was started in the month of November it was quite unsure that bat consumption should be considered. As the size of bats are very small and generally not preferred for market. Now we found many evidence of poaching and smuggling of Pangolin(_Manis javanica_) in the area from China to Myanmar `(Hassanin et al. 2021)`. It can be another point which we can consider. Pangolin are the mammals which are believed to live separately so it is hard that they might get the infection in a crowd rather than one of theory suggest it is possible while smuggling they came in contact with Bat. Another theory suggests that they might have eaten the flesh of bats from the cave. Through smuggling it reaches a wet market and finally the consumption of Pangolin makes it  enter the Human. Throughout this paper we will be discovering many aspects on how the virus breaks and from where it originally originated. We will also consider the available data for Sarbecoviruses(virus related to SARS CoV and SARS CoV-2) to propose all the hypotheses related to the Origin of COVID-19. Apart from discovering its origin we will also look at how much it is related with the present Human Coronavirus strains. Further we will study about the important component of this virus which is of course the pathogenesis of it. We will look into how much it is different from the other coronavirus in terms of mutation. At last we will see how we can identify important signatures that can significantly different under therapeutic(drug)  treatment and untreated condition


__Keywords__: bats; covid-19; pangolins; SARS- CoV-2; virus;  wet market.


![Logopit_1615992905323](https://user-images.githubusercontent.com/63470232/111488351-29ed6a00-875f-11eb-87dc-32f696bda8dc.jpg)




## Content
 - [Introduction](#Introduction)
 - [SARS CoV 2 genome](#SARSCoV2genome)
 - [Zoonotic origin](#zoonotic_origin)
   * [Phylogenetic Analysis](#zPhy)
   * [Multiple Sequence Alignment](#zMul)
 - [Pathogenesis](#pathogenesis)
 - [Conclusion](#conclusion)
 - [Reference](#reference)



## Introduction <a name="Introduction"></a>

__Coronavirus__ are the viruses known for having the longest genome of all the known RNA- viruses. It has a crown like structure, from where it is given the name corona(is a latin word of crown). These are the spike proteins on the exterior. Previously this virus was known for mostly affecting Birds or small mammals but as we can see now it has the ability to have human to human transmission. In humans and Birds, they cause respiratory tract infection, this can be mild, but can be lethal at some instances`(Fouchier et al. 2004)`. Mild illness shows the symptoms of Cold, many articles claim that the cold having dry cough are the most out of them. But there are many exceptional cases already present which show the number of patients having wet coughs and still infected. On rare occasions, gastrointestinal coronavirus infection has been associated with the outbreak of diarrhoea in children`(Tyrrell and Myint 2011)`.

![Screenshot 2021-03-14 at 9 05 34 AM](https://user-images.githubusercontent.com/63470232/112012627-5cc4a300-8b4f-11eb-8d49-3630fe7524bd.png)

Based on the cell that is likely infected it is classified into three different phases corresponding to different stages of the disease`(Wu and McGoogan 2020)`.The early phase of Asymptotic state[__initial 1-2 days__] where SARS-CoV-2 inhaled virus binds to epithelial cells in the nasal cavity and begins replicating. __SARS-CoV2__ and __SARS-CoV__ both use the __ACE2__ receptor . The ciliated cells are main cells infected in the conducting airways, according to in vitro data with SARS-CoV. The virus is circulating locally, but there is only a small innate immune response. Nasal swabs will diagnose the virus at this point.These people are contagious, even though their viral burden is minimal. The viral RNA __RT-PCR__ value can be useful in predicting viral load, eventual infectivity, and clinical path.These experiments could be able to detect super spreaders`(Mason 2020)`. The sample selection protocol will have to be standardised for the RT-PCR period number to be meaningful. Swabs from the nose can be more susceptible than swabs from the throat.

In the second phase The virus spreads and migrates down the respiratory tract, causing a more intense innate immune response.The virus (SARS-CoV-2) as well as initial markers of the innate immune response can be included in nasal swabs or sputum. The disordered __COVID-19__ is now scientifically manifest. __CXCL10__ (or another innate response cytokine) level can be representative of potential clinical outcomes`(Sims et al. 2005; Hoffmann et al. 2020)`. Beta and lambda interferons are formed in large amounts by virally infected epithelial cells. __CXCL10__ is an interferon-responsive gene that has a high signal-to-noise ratio in the response of alveolar type II cells to __SARS-CoV__ and __influenza__. __CXCL10__ has also been proposed as a disease marker in the case of SARS. Determining the host's innate immune response may help foresee the disease's future path and the need for more active control.

But most of the people enter the third stage and develop pulmonary infiltrates, with some developing life-threatening disease. Its response varies with the age group. Once the prevalence of mild and asymptomatic cases is better described, the fatality and morbidity rates can be updated.The virus has now infiltrated alveolar type II cells in the lung's gas exchange units. In comparison to type I cells, __SARS-CoV__ and influenza preferentially infect type II cells. The infected alveolar units are usually found on the periphery and in the centrilobular space. `(Okada et al. 2007; Hoffmann et al. 2020; Mason 2020)`__SARS-CoV__ replicates in type II cells, releasing a large number of viral particles before the cells die of apoptosis. As far age is concerned because of their weakened immune responses and decreased ability to repair damaged epithelia, the elderly are especially vulnerable. The elderly also have lower mucociliary clearance, which could make it easier for the virus to spread to the lung's gas exchange units.

We understand the different stages of the transmission as well as replication of the virus, these viruses are normally spread by droplets in the air (coughing, sneezing). Oral-fecal transmission experiments have recently been launched to see whether viral shedding from asymptomatic patients can induce transmission without coughing or sneezing. Virus transmission and the risk of transmission without direct contact have several aspects.

![spiral Chart](https://cmr.asm.org/content/cmr/28/2/465/F7.medium.gif)

## SARS CoV 2 genome <a name="SARSCoV2genome"></a>

![Spiral Chart](https://www.mdpi.com/pathogens/pathogens-09-00331/article_deploy/html/images/pathogens-09-00331-g001.png)

The virus is really not alive; it's just a chunk of grease with a chromosome within. Only a compatible host would be able to use this genome or in other sense we can say it will infect or can survive in a particular specific host. As a result, the virus's genome(a sequence of letters containing all of the knowledge) required to replicate and assemble a virus, is at its heart. Before we begin searching for this sequence, it's important to note that COVID-19 is caused by a particular type of virus known as a "positive strand RNA virus" based on the way this information is collected.

SARS-COV2 is a single-strand RNA virus, which means it doesn't store the genetic material in DNA but instead in a single strand of RNA. Since RNA viruses have a higher mutation rate than DNA viruses, they are the most prevalent cause of emerging diseases in humans.As compared to RNA, DNA is a much more stable macromolecule. RNA viruses use virally encoded RNA-dependent RNA polymerase to copy their genomes __(RdRp)__.The RNA genome serves as a blueprint for the development of new RNA strands. At least three types of RNA must be synthesised during RNA virus replication: the genome, a copy of the genome (reference genome), and mRNAs. Some RNA viruses, along with coronaviruses, make subgenomic mRNA copies as well. Positive-sense RNA viruses (abbreviated __+ssRNA__) are viruses with __ssRNA__ genomes that can act as mRNA directly. Negative-sense RNA viruses, on the other hand, are ssRNA viruses with genomes that cannot be directly translated by ribosomes and are instead used as a basis for mRNA processing (abbreviated -ssRNA). __+ssRNA__ viruses infect a wide variety of host species and are more widespread than any other virus family. These include seven separate human viral families that cause major disease in humans, such as coronaviruses, flaviviruses, and picornaviruses`(Crowe 2004; Desforges et al. 2019; Firth et al. 2014)`. Since the genome of __+ssRNA__ viruses functions as __mRNA__, they provide contagious genetic material. Their genomes have 5′-caps (or proteins that behave similarly to a 5′-cap) and poly(A) tail sequences at the 3′-end, and are translated by host ribosomes.

## Zoonotic Origin <a name="zoonotic_origin"></a>

Whenever any pandemic occur, researcher generally look to answer the following question: What was the source of this new coronavirus? What species, when did it first infect humans, and how did it do so? This all question answer will further help the researcher designing the vaccine. Our target in this objective is to determine its zoonotic origins, or which animal it came from. To answer this query, we'll need to look for data structures in the genomes of corona viruses that can help us answer it.

### Phylogenetic Analysis <a name="zPhy"></a>

The use of a phylogenetic analysis technique will help us figure out where the novel SARS-COV-2 virus came from. To do so, we compared the SARS-COV-2 genome sequences to those of other coronavirus strains from different species in a phylogenetic tree. Since the pandemic began in China. We want to compare strains from human SARS-COV-2, bat coronavirus, and pangolins using China as a "geological spot."

We collected data from NCBI With fixing the origin to China, and to make the plot look comfortable we take few genome sequence two- three each from our classes. We have taken Homosapience as reference genome. 

![Screenshot 2021-02-13 at 1 54 33 PM](https://user-images.githubusercontent.com/63470232/112143686-3dd11a00-8bfe-11eb-92aa-49ff82f3f581.png)

When looking at the phylogenetic trees produced by the study, it's clear that the bat coronavirus is the earliest in the series, and that it's possible that it mutated to produce Human SARS-COV2 and Pangolin coronavirus. Furthermore, the fact that SARS-COV2 and pangolins are in similar clades suggests that they have closer genetic relationships. Since pangolin coronavirus and SARS-COV-2 have similar ancestral relationships, these findings suggest that SARS-COV-2 evolved from pangolin coronavirus.

### Multiple Sequence Alignment <a name="zMul"></a>

The next step is to figure out whether the pangolin coronavirus resembles SARS-COV2 rather than the Bat coronavirus. Multiple alignments may be performed and visualised. Let's imagine MSA in the UGENE now that we've collected it from our pipeline.Bat coronavirus has significantly more variance or mutations than SARS-COV2 and Pangolin coronavirus. 

The MSA of Human SARS-COV2, Bat-SARS, Bat coronavirus, and Pangolins coronavirus represents the variations in sequences at the Ammino Acid level. Human SARS-COV-2 and pangolin coronavirus have a 7-9 percent dissimilarity, whereas Human SARS-COV-2 and Bat coronavirus have a 15-17 percent dissimilarity.It also implies that not all variants at the nucleotide level are passed to the Ammino Acid level, since the nucleotide level has a higher percent age with variations than the Amino Acid level. Furthermore, SARS-COV-2 is more similar to the pangolin coronavirus than the Bat-SARS and Bat coronavirus, similar to the pattern at the nucleotide level variation`(Zhang et al. 2020)`. 

## Pathogenesis <a name="pathogenesis"></a>

SARS-COV-2 is enclosed by a lipid bilayer membrane, similar to how a ball of grease is surrounded by a lipid bilayer membrane.The membrane protein (M protein), the envelope protein (E protein), and the spike protein(S protein) make up the lipid bilayer or viral envelope.The interaction between the virus's Spike protein and the host's Angiotensin-Converting Enzyme 2 (ACE2) receptor is critical for pathogenesis, according to published research.

![](https://cdn.britannica.com/47/215947-050-66A6BF8B/Severe-acute-respiratory-syndrome-Coronavirus-SARS-CoV-2-COVID-19.jpg)

Coronavirus is also famous for its crown like structure, they are due to the spike  glycoprotein on the outer surface connecting the interior and are the main position for interacting with any living cell. Spike protein is a glycosylated transmembrane fusion protein that varies in size between coronaviruses. The S-protein, which has a length of ~1200 amino acids, is a class-I viral fusion protein that plays a role in cell receptor binding, tissue immune response, and pathogenesis. The spike protein is important for infecting the host cell and penetrating it. Spike protein is a trimer protein with two subunits of each monomer.(Lin et al., 2020)Specifically, the S1 subunit at the N-terminus and the S2 subunit at the C-terminus. There are two cleavage sites in the coronavirus spike protein. During infection, host cell proteases process the S1/S2 cleavage site. In some viruses, the presence of a furin-like S2' second cleavage site between the fusion peptide (FP) and the internal fusion peptide (IFP) has been shown to enhance pathogenicity and cellular tropism. Proprotein convertases (PCs) cleave and mediate the pathogenicity of precursor viral proteins including Spike glycoprotein. By improving viral fusion with host cells, the cleaving and processing of essential cell surface proteins, especially furin, helps to prevent viral infections. SARS-CoV2 and SARS both have this functionality. As the S1 subunit binds to the host cell receptor, the S2 subunit undergoes a significant conformational transition. This shift is critical for fusion. HR1 and HR2 are two heptad repeat regions that make up the S2(Fouchier et al., 2004; Desforges et al., 2019; Mason, 2020). The unstructured, pre-hairpin linker S2 structure becomes helical in the first conformational transition. The S2 C-helix turns to a coil in the second conformational transition, gradually forming a six-helix bundle. The anchoring of the fusion peptide to the host cell membrane, and hence the delivery of the nuclear material into the host cell, is dependent on these conformational changes.

![Picture27](https://user-images.githubusercontent.com/63470232/112423661-16429480-8d59-11eb-866b-75f1e7a9d1c2.png)


We discovered that the RBD regions of Pangolin coronavirus have the fewest variations, while MERS has the most variations as compared to SARS-COV-2. This most likely suggests their pathogenicity. 

## Conclusion <a name="conclusion"></a>

Based on the evidence obtained from 11 sample strains from NCBI, phylogenetic analysis and multiple sequence alignment (MSA) specifically reveal that Pangolin coronavirus is more similar to Human SARS-COV2. Pangolin Coronavirus is most likely the source of Human SARS-COV-2. This also suggests another aspect apart from medical research that there is a large number of poaching cases going on for pangolin and other wildlife creatures. 

## Reference <a name="reference"></a>

Crowe, J. E. (2004) ‘Human Metapneumovirus as a Major Cause of Human Respiratory Tract Disease’, Pediatric Infectious Disease Journal, pp. S215–S221. doi: 10.1097/01.inf.0000144668.81573.6d.

Desforges, M. et al. (2019) ‘Human Coronaviruses and Other Respiratory Viruses: Underestimated Opportunistic Pathogens of the Central Nervous System?’, Viruses, 12(1). doi: 10.3390/v12010014.

Desmarest, A. G. (1822) Mammalogie ou description des espèces de mammifères. chez Mme veuve Agasse. Available at: https://play.google.com/store/books/details?id=oUs_AAAAcAAJ.

Covid 19 origin and pathogenesis Available at:https://edu.t-bio.info/course/covid-19-origin-and-pathogenesis-of-sars-cov2/lessons/coronaviruses-introduction/

Firth, C. et al. (2014) ‘Detection of zoonotic pathogens and characterization of novel viruses carried by commensal Rattus norvegicus in New York City’, mBio, 5(5), pp. e01933–14. doi: 10.1128/mBio.01933-14.

Fouchier, R. A. M. et al. (2004) ‘Avian influenza A virus (H7N7) associated with human conjunctivitis and a fatal case of acute respiratory distress syndrome’, Proceedings of the National Academy of Sciences of the United States of America, 101(5), pp. 1356–1361. doi: 10.1073/pnas.0308352100.

Hassanin, A., Grandcolas, P. and Veron, G. (2021) ‘Covid-19: natural or anthropic origin?’, Mammalia, 85(1), pp. 1–7. doi: 10.1515/mammalia-2020-0044.
Hoffmann, M. et al. (2020) ‘SARS-CoV-2 Cell Entry Depends on ACE2 and TMPRSS2 and Is Blocked by a Clinically Proven Protease Inhibitor’, Cell, pp. 271–280.e8. doi: 10.1016/j.cell.2020.02.052.

Lin, L. et al. (2020) ‘Hypothesis for potential pathogenesis of SARS-CoV-2 infection–a review of immune changes in patients with viral pneumonia’, Emerging microbes & infections, 9(1), pp. 727–732. doi: 10.1080/22221751.2020.1746199.

Mason, R. J. (2020) ‘Pathogenesis of COVID-19 from a cell biology perspective’, The European respiratory journal: official journal of the European Society for Clinical Respiratory Physiology, 55(4). doi: 10.1183/13993003.00607-2020.

Morens, D. M. et al. (2020) ‘The origin of COVID-19 and why it matters’, The American journal of tropical medicine and hygiene, 103(3), pp. 955–959. doi: 10.4269/ajtmh.20-0849.

Okada, F. et al. (2007) ‘Clinical/pathologic correlations in 553 patients with primary centrilobular findings on high-resolution CT scan of the thorax’, Chest, 132(6), pp. 1939–1948. doi: 10.1378/chest.07-0482.

Sims, A. C. et al. (2005) ‘Severe acute respiratory syndrome coronavirus infection of human ciliated airway epithelia: role of ciliated cells in viral spread in the conducting airways of the lungs’, Journal of virology, 79(24), pp. 15511–15524. doi: 10.1128/JVI.79.24.15511-15524.2005.

Tyrrell, D. A. J. and Myint, S. H. (2011) ‘Coronaviruses’, in Baron, S. (ed.) Medical Microbiology. Galveston (TX): University of Texas Medical Branch at Galveston. Available at: https://www.ncbi.nlm.nih.gov/pubmed/21413266.

Wu, Z. and McGoogan, J. M. (2020) ‘Characteristics of and important lessons from the coronavirus disease 2019 (COVID-19) outbreak in China: summary of a report of 72 314 cases from the Chinese Center for Disease Control and Prevention’, JAMA: the journal of the American Medical Association, 323(13), pp. 1239–1242. Available at: https://jamanetwork.com/journals/jama/article-abstract/2762130.

Zhang, T., Wu, Q. and Zhang, Z. (2020) ‘Probable Pangolin Origin of SARS-CoV-2 Associated with the COVID-19 Outbreak’, Current biology: CB, 30(7), pp. 1346–1351.e2. doi: 10.1016/j.cub.2020.03.022.
